Composer/packages/client/src/components/CreationFlow/CreationFlow.tsx

```diff
@@ -150,43 +159,61 @@ const CreationFlow: React.FC<CreationFlowProps> = () => {
     navigate(`./create/${data}`);
   };
 
+  const pluginConfig: PluginConfig = useMemo(() => {
```

Comment:
this can be a const outside the component code, right?

Reply:
I believe since it uses the `useMemo` hook it has to be declared in the functional component